src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -46,24 +46,24 @@
 
 helps['webapp identity assign'] = """
     type: command
-    short-summary: (PREVIEW) assign managed service identity to the webapp
+    short-summary: assign or disable managed service identity to the webapp
     examples:
         - name: assign local identity and assign a reader role to the current resource group.
           text: >
-            az webapp identity assign -g MyResourceGroup -n MyUniqueApp --role reader --scope /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/MyResourceGroup
```

Comment:
i suggest you keep the `--role` and `--scope` , as otherwise this webapp  can't do anything really with the identity, also the description 2 lines above still calls out using the `role`